feature.env

```diff
@@ -7,6 +7,7 @@ OFFICIAL_VISITS_API_URL=http://localhost:9091/official-visits-api
 PRISON_API_URL=http://localhost:9091/prison-api
 PERSONAL_RELATIONSHIPS_API_URL=http://localhost:9091/personal-relationships-api
 COMPONENT_API_URL=http://localhost:9091/components
+ACTIVITIES_API_URL=http://localhost:9091/activities-api
 DPS_HOME_PAGE_URL=http://localhost:9091
```

Comment:
Perhaps send a reminder to the other devs (when this is merged) to add ACTIVITIES_API_URL to their .env files locally?